### ISSUE TITLE
Add nullability annotation for primitive setters

### ISSUE THREAD

**Issue by pronebird** [rentzsch/mogenerator#361]
### Feature Request

Nullability annotations are not added for primitive setters, i.e.:

```objc
@property (nonatomic, strong, nullable) Foo *foo;

- (void)setPrimitiveIncome:(Foo*)value;
```

### Expected Behavior

Primitive setter should be generated with nullability annotation.

```objc
- (void)setPrimitiveFoo:(__nullable Foo*)value;
```

### Actual Behavior

Nullability annotation is not being added to method declaration.

```objc
- (void)setPrimitiveIncome:(Foo*)value;
```

### Additional Information

Mogenerator v1.31

**Comment by mbabin** [comment]
More specifically, I believe nullability annotation is not added for primitive accessors of optional, to-one relationships, leading to a mismatch with the corresponding property accessors.

**Comment by atomicbird** [comment]
This was fixed in #363, which has just been merged.
